src/controllers/game.controller.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from "express";
+import { ZodError } from "zod";
+import {
+  gameCreationSchema,
+  gameUpdateSchema,
+} from "../validation/gameValidation.schema";
+import { GameModel } from "../models/game.model";
+import { AuthenticatedRequestParams } from "../interfaces/authenticatedRequestParams";
+
+export const createGame = async (req: Request, res: Response) => {
+  try {
+    gameCreationSchema.parse(req.body);
+    const createdGame = await GameModel.create({
+      team1: req.body.team1,
+      team2: req.body.team2,
+      startTime: req.body.startTime,
+      scoreTeam1: 0,
+      scoreTeam2: 0,
+      pointDifference: 0,
```

Comment:
quid set this as null as long as we haven't set the score ourselves so it's easier to see which games have been completed or not ?

Reply:
Yes ! Already changed it in the compute points PR https://github.com/Vico4/bipboup-api/pull/12/files :)